Realign action button on view properties screen

bryangaledxw opened the pull request:
# Changes in this PR

- Aligns action button on property listing with heading

## Screenshots of UI changes

### Before
![localhost_3000_properties (5)](https://user-images.githubusercontent.com/94137563/213482246-d02502f3-0167-45c7-9e79-79973e0a57d2.png)

### After
![localhost_3000_properties (4)](https://user-images.githubusercontent.com/94137563/213482251-decc36d7-9ee8-4e7e-91ee-9a4aff54a57b.png)

# Release checklist

As part of our continuous deployment strategy we must ensure that this work is
ready to be released at any point. Before merging to `main` we must first
confirm:

## Pre merge checklist

- [ ] Have all changes to any dependencies been deployed to both `preprod` and
    `production` in a backwards compatible way? (This includes our API,
    infrastructure, third-party integrations and any secrets or environment variables)
- [ ] Has a required data migration been included in this change or been done in
    advance?

## Post merge checklist

Once we've merged we now need to release this through to production before
considering it done.

[Find the build-and-deploy job in CircleCI](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-temporary-accommodation-ui)
and work through the following steps:

- [ ] Has the product manager asked to provide approval for this feature?
  - [ ] Has the product manager approved?
- [ ] Manually approve release to preprod
- [ ] Verify change released to preprod
- [ ] Manually approve release to prod
- [ ] Verify change released to production

Should a release fail at any step, you as the author should now lead the work to
fix it as soon as possible.

bryangaledxw commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Realign action button on 'view properties' screen.](https://trello.com/c/tE98UOUH/705-realign-action-button-on-view-properties-screen)

tahb reviewed (commented):
This looks good on wider screens but on smaller screens (~ <1200px) the header gets pushed right by a container margin, this makes it look a little odd where the header is no longer inline with the content.

 ![Screenshot 2023-01-23 at 12 23 19](https://user-images.githubusercontent.com/912473/214038990-3cad05f7-2115-401b-92ab-f10bdc0714fe.png)

I think we might want the [Page header actions component](https://design-patterns.service.justice.gov.uk/components/page-header-actions/) here instead? It thankfully appears to fix this problem [in their example](https://design-patterns.service.justice.gov.uk/examples/page-header-actions/).

bryangaledxw committed:
Update colour specification

We use a provided GDS macro, rather than a hardcoded colour value

bryangaledxw committed:
Update CSS for page header bar

Following the move from the identity bar to the page header bar, we update our CSS. We add an additional top margin to the header bar, and use a 1px shadow to restore the horizontal line we lost on the booking page when we switched component

bryangaledxw committed:
Use MoJ page header bar

We replace the use of the MoJ identity bar component, with the more appropriate MoJ page header bar component. The necessitates some minor integration test changes, to allow Cypress to find the action buttons. We also need some changes in bookingUtils, as the action buttons are populated from a list of items, rather than a list of lists of items

bryangaledxw committed:
Remove unused import from booking history template

bryangaledxw committed:
Use MoJ page header bar on property listing page

This allows us to better align the action button with the header text

tahb reviewed (approved):
LGTM 👍
